app/javascript/crayons/navigation/NavigationTabs/__stories__/navigationTab.html.stories.jsx

```diff
@@ -9,13 +9,17 @@ export default {
 
 export const Default = () => (
```

Comment:
(suggestion / not blocking) - nice one updating the storybook component too! I think it would be worth adding a note to `navigation-tab.md` to explain why we add the `data-text` attribute too

Reply:
Very good point!